src/GitHub.Api/ApiClientConfiguration_User.cs

```diff
@@ -1,11 +1,13 @@
+﻿using System;
+
 namespace GitHub.Api
 {
-    public partial class ApiClient : IApiClient
+    static partial class ApiClientConfiguration
```

Comment:
A separate PR that fixes a few things for public builds would certainly be good. We could sort out the public / open source signing issue at the same time. Recent versions need to be signed and this isn't happening at the moment. They're all related issues with quite similar fixes.